docs/reference/changelog.rst

```diff
@@ -2,6 +2,15 @@
 Changelog
 *********
 
+X.Y.Z (2024-MM-DD)
```

Comment:
Are you going to need a release of craft-platforms this pulse?

Reply:
Yep, this snapcraft change will require a new craft-platforms release.

Reply:
Yep that's fine.  But before we release craft-platforms let me finish the snapcraft change so I can be sure no more c-plat changes will be needed.

Reply:
Don't need to make more changes to c-plat, we're all set on this one

Reply:
Change this to 0.4.0 with the date 2024-10-17 and I'll release in the morning!

Reply:
Excellent, will do thanks!